lancet/cli.py

```diff
@@ -32,7 +33,15 @@
 
 
 def get_sentry_client(sentry_dsn):
-    return sentry_sdk.Client(sentry_dsn, release=__version__)
+    try:
+        return sentry_sdk.Client(sentry_dsn, release=__version__)
+    except sentry_sdk.utils.BadDsn:
+        click.secho(
+            f'Please provide a valid sentry_dsn value in "{USER_CONFIG}"',
+            fg="red",
+            bold=True,
+        )
+        click.secho()
```

Comment:
When does this happen? Might be worth to raise an `Abort` exception instead.

Reply:
switch to exit with proper text, seems more clean and added comments :)

Reply:
Had to revert cause aborting prevents the file to be rewritten using lancet setup -f. The warning should show it a command is run and sentry_dsn is incorrectly configured

Reply:
Why would it prevent it? Also, use an exception instead of sys.exit; easier to handle and easier to test for.

Reply:
@GaretJax any command that runs, if there is an incorrect sentry_dsn added will fail, even if I want to re-setup it again using ``lancet setup -f`` resulting into editing the file manually to fix the issue if I hard abort (exit, or raise exception). That's why I rather have it as a Warning rather than aborting the command.